25.js

```diff
@@ -0,0 +1,157 @@
+// Create class UserService
+// Implement every scenario from dummy users BE in your class
+// For each scenario create new method
+// Methods can be static
+// Read response from BE and console.log it or store it in class properties
+// Don't forget to catch errors when request failed
+
+// *Extra task: Delete users with ids 1,2,3,4 and random id from 5 to 10000 simultaneously
+// Use Promise.allSettled for this task
+
+class UserService {
+  users = [];
+  selectedUser = null;
+  searchedUsers = [];
+  filteredUsers = [];
+  carts = null;
+  posts = null;
+  todos = null;
+  page = 0;
+
+  getAllUsers() {
+    fetch("https://dummyjson.com/users", {
+      method: "GET",
```

Comment:
GET is default method so there is no need to specify it explicitly

Reply:
I had problems with requests,so I added the method GET just in case